src/core/common/inviwoapplication.cpp

```diff
@@ -128,6 +128,7 @@ InviwoApplication::InviwoApplication(int argc, char** argv, std::string displayN
     , propertyConverterManager_{std::make_unique<PropertyConverterManager>()}
     , propertyFactory_{std::make_unique<PropertyFactory>()}
     , propertyWidgetFactory_{std::make_unique<PropertyWidgetFactory>()}
+    , representationMetaFactory_{std::make_unique<RepresentationMetaFactory>()}
```

Comment:
missing initialization of representationFactories_ and functors?

Reply:
that done by the modules, Core and OpenGL